README.md

```diff
@@ -79,6 +79,35 @@ julia> convert(Vector{FK5Coords{1975, Float64}}, c1)
  SkyCoords.FK5Coords{1975,Float64}(6.2776,-0.00242922)
 ```
 
+### Angular Separation between Coordinates
+
+The `separation` function allows you to compute the angular (great-circle)
+distance between two coordinates, in radians, using
+the [Vincenty formula](http://en.wikipedia.org/wiki/Great-circle_distance).  The
+coordinates can be also given in different systems.  For example, according to
+SIMBAD the `FK5Coords{2000}` coordinates
+of [Mizar](http://simbad.u-strasbg.fr/simbad/sim-id?Ident=MIZAR) are
+
+```julia
+julia> mizar = FK5Coords{2000}(3.507787, 0.958628)
+```
+
+while the `GalCoords` coordinates of Alcor are
+
+```julia
+julia> alcor = GalCoords(1.968189, 1.072829)
+```
+
+Their angular separation is given by
+
+```julia
+julia> separation(mizar, alcor) # Radians
+0.003435309169452688
+
+julia> separation(mizar, alcor)*3600 # Arcseconds
```

Comment:
Remember me not to do calculation after 1 AM next time ;-)

However, now I'm a bit confused by the result:

```julia
julia> rad2deg(separation(mizar, alcor))*3600
708.4952936319647
```

Isn't this a bit too much?  I didn't catch the error because 12 is more or less the expected separation between the two stars.  I calculated the coordinates with:

```julia
mizar = FK5Coords{2000}(round(deg2rad(ten(13,23,55.54)*15), 6), round(deg2rad(ten(54, 55, 31.27)), 6))
alcor = GalCoords(round(deg2rad(112.7689), 6), round(deg2rad(+61.4686), 6))
```

(`ten` is from `AstroLib.jl`).  What am I missing?